utils/create-breadcrumbs.ts

```
@@ -2,85 +2,186 @@
 
-    if (frontMatter.description &&
-        frontMatter.description.match(/\bOP Mainnet\b|\bOptimism Mainnet\b/gi) &&
-        !frontMatter.description.includes('OP Stack')) {
+  const overviewFile = files.find(file => 
+    file.url.toLowerCase().includes('overview') || 
+    file.url.toLowerCase().includes('introduction')
+  );
 
-      frontMatter.description = updateOPTerminology(frontMatter.description);
-      
+  if (overviewFile && overviewFile.content) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)